scripts/src/signedchart/signedchart.py

```diff
@@ -41,10 +42,12 @@ def get_verifier_flags(tar_file, owners_file, temp_dir):
 def is_chart_signed(api_url, report_path):
     if api_url:
         files = prartifact.get_modified_files(api_url)
-        tgz_pattern = re.compile(r"charts/(\w+)/([\w-]+)/([\w-]+)/([\w\.-]+)/.*.tgz")
+        tgz_pattern = re.compile(
+            matchers.submission_path_matcher(strict_categories=False) + r".*.tgz"
+        )
         tgz_found = False
         prov_pattern = re.compile(
-            r"charts/(\w+)/([\w-]+)/([\w-]+)/([\w\.-]+)/.*.tgz.prov"
+            matchers.submission_path_matcher(strict_categories=False) + r".*.tgz.prov"
```

Comment:
It's great that you paid attention to backward compatibility with this change. Part of me wonders if things break if we turn `strict_categories` to True here. I would imagine it doesn't and we can simplify this a little bit by getting rid of this argument.

Reply:
I had the same thought but didn’t want to find out 😂 in my mind, it would seem like having less strict category could be a thing in some  circumstances, like if a contributor were to typo a category some how… but I didn’t chase it down. I can certainly try it.

Reply:
I might aim to merge this as is so I can include  this in the release because of the fact that the current changes do not solve the plus sign versioning issue sufficiently, which this is trying to help.

Reply:
I think I landed on wanting this type of evaluation to happen in a later change. If that's workable for you, please mark this resolved.

Reply:
Absolutely fine with me !